app/scripts/migrations/045.js

```diff
@@ -0,0 +1,23 @@
+const version = 45
+import { cloneDeep } from 'lodash'
+
+/**
+ * Replace ipfsGateway by 'dweb.link'.
+ */
+export default {
+  version,
+  migrate: async function (originalVersionedData) {
+    const versionedData = cloneDeep(originalVersionedData)
+    versionedData.meta.version = version
+    const state = versionedData.data
+    versionedData.data = transformState(state)
+    return versionedData
+  },
+}
+
+function transformState (state) {
+  if (state.PreferencesController && state.PreferencesController.ipfsGateway) {
```

Comment:
@hacdias I imagine we want to only override if value is `ipfs.dweb.link`?
```suggestion
  if (state.PreferencesController && state.PreferencesController.ipfsGateway === 'ipfs.dweb.link') {
```

Reply:
Hmm.. I thought about changing it anyways because most setups will be broken if we keep a custom setting. However, by breaking those set ups, people will be aware and change it by themselves...

Reply:
```suggestion
  if (state.PreferencesController?.ipfsGateway) {
```